playbooks/robusta_playbooks/pod_investigator_enricher.py

```diff
@@ -144,6 +153,7 @@ def report_pod_issue(event: KubernetesResourceEvent, pods: List[Pod], issue: Pod
         blocks.append(MarkdownBlock(f"\n\n*{pod_names[0]}* was picked for investigation\n"))
         blocks.extend(additional_blocks)
         event.add_enrichment(blocks)
+        event.add_enrichment(MarkdownBlock(f"\n\nProblem cause: {issue_details}"))
```

Comment:
`add_enrichment` gets an array of blocks.
Should be `event.add_enrichment([MarkdownBlock(f"\n\nProblem cause: {issue_details}")])`

Reply:
Well, yeah. Funny things this has been verified to work :D

Reply:
fixed